src/expr/relation/mod.rs

```diff
@@ -117,6 +135,12 @@ pub enum RelationExpr {
         implementation: JoinImplementation,
     },
     /// Group a dataflow by some columns and aggregate over each group
+    ///
+    /// The runtime memory footprint of this operator is at most proportional to the
+    /// number of distinct records in the input and output. The actual requirements
+    /// are reduced to the number of distinct inputs to each aggregate, summed across
```

Comment:
minorly confusing non-`Reduce`-related usage of "reduce" in documentation for `Reduce`

Reply:
Language improved. Thanks!